Tests/Functional/.lit_test_times.txt

```diff
@@ -0,0 +1,25 @@
+1.238145e+01 Performance/main.swift
```

Comment:
I don't think we want to include this file, do we? The other PR included a change to a .gitignore file to ignore this… Should we merge these two back-porting PRs into one so the .gitignore change will take effect also?

Reply:
Did I accidentally commit this?  I'm sorry, this wasn't meant to be added.